<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homographyForm</class>
 <widget class="QWidget" name="homographyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>596</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="3,7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0,0,0,0,1,1">
         <item>
          <widget class="QPushButton" name="loadImageButton">
           <property name="text">
            <string>Load Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="undistortButton">
           <property name="text">
            <string>Undistort</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="improvePointsButton">
           <property name="text">
            <string>Improve position of points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="calculateHomographyButton">
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveHomographyButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mainMenuButton">
           <property name="text">
            <string>Main menu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,0,0">
         <item>
          <widget class="QComboBox" name="selectSportBox"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="widthLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="widthLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="heightLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="heightLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="sizeButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearPointsButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="pointsScaleFactorLine">
             <property name="text">
              <string>Points Scale Factor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pointsScaleFactorEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="showBirdsEyeBox">
           <property name="text">
            <string>Show Birds Eye View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="mappingLayout"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="imageLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
